bot.py

```diff
@@ -85,6 +84,17 @@ async def off_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
     else:
         await update.message.reply_text("You don't have an active session. Use /on to start the timer.")
 
+async def abort_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    user_name = update.effective_user.username
+    if user_name in user_sessions:
+        if user_name == "Unknown":
+            await update.message.reply_text("You are not registered yet. Contact @samtjong to register before you can use this bot.")
+        else:
+            del user_sessions[user_name]
+            await update.message.reply_text("Your session has been cancelled. Use /on to start a new timer.")
+    else:
+        await update.message.reply_text("You don't have an active session. Use /on to start the timer.")
```

Comment:
Nitpick: change to 'start a new timer'

Reply:
[Resolved!](https://github.com/samtjong23/airconTeleBot/pull/1/commits/192824c4600a245ad8a0bb2eefc183f51a533d51)